benchmarks/dynamo/common.py

```diff
@@ -839,6 +842,39 @@ def baselines(models, model_iter_fn, example_inputs, args):
     return result
 
 
+def xla(args, model_iter_fn, model, example_inputs):
+    xla_dev = xm.xla_device(devkind=current_device)
+    model_xla = copy.deepcopy(model).to("cpu").to(device=xla_dev)
```

Comment:
why to cpu first?

Reply:
Throws an error on direct cuda -> xlacuda.

Reply:
what's the perf implication of this line?

Reply:
Should be zero since it done once on startup before any performance measurement.